packages/vscode-extension/e2e_tests/fixtures/.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "neo4j.connect": true,
+  "neo4j.password": "password",
+  "neo4j.URL": "neo4j://localhost:55161",
+  "neo4j.user": "neo4j",
```

Comment:
This seems like an unrelated change, is it to make the debugging easier?

Reply:
Sorry, this is part of the other pr. It will allow us to have e2e tests for VSCode with customizable language server configuration